pokedex-with-bit/app.py

```diff
@@ -25,20 +24,54 @@ def config():
 
 
 def index(num_docs: int):
-    f = Flow.load_config('flows/index.yml')
-    num_docs = min(num_docs, len(glob(os.path.join(os.getcwd(), IMAGE_SRC), recursive=True)))
-    with f:
-        with TimeContext(f'QPS: indexing {num_docs}', logger=f.logger):
-            f.index_files(IMAGE_SRC, request_size=64, read_mode='rb', size=num_docs)
+    num_docs = min(num_docs, len(glob(os.path.join(os.getcwd(), IMAGE_SRC),
+                                      recursive=True)))
 
+    flow = Flow(workspace="workspace")\
```

Comment:
Not a strong opinion but it may improve readability to extract the Configs to the other global Variables above.

Reply:
Maybe you can reuse some of the configs for the query Flow

Reply:
I actually prefer writing it explicitly in both functions. I agree there is some code duplication but it is only minor.
Also, I think it is better for understanding if the flows are configured in one place only ant not distributed in different functions.